### DISCUSSION TITLE
TS0601 _TZE204_7rghpoxo external defenition

### DISCUSSION THREAD

**Discussion by avgorsh** [Koenkk/zigbee2mqtt#25376] [Devices]
Hi Everyone,
I am trying to add a new device that is currently not supported.
The device does not expose anything in Z2M.
I can only see clusters there.
I have added second piece to Tyua server and have access to Tyua developer account.
There one can find some documentation there.
Can the documentation be used to create external definition?
Can this documentation simplify creating the definition?

I am quite new to the topic thus trying to find the best approach.
At this moment mapping of datapoints looks like a challenge.

Example of docs:
![image](https://github.com/user-attachments/assets/cf04857a-fb67-4466-8114-a4cb37abe1ec)
I can also see some results for my requests:
{
  "result": {
    "active_time": 1734247658,
    "bind_space_id": "213566409",
    "category": "wk",
    "create_time": 1734247658,
    "custom_name": "",
    "icon": "smart/icon/bay1592043850975a2gH/8ff49ed1f4286d07394a63abad96d9e5.png",
    "id": "bf4c4ed80df33070191",
    "ip": "",
    "is_online": true,
    "lat": "0",
    "local_key": "",
    "lon": "0",
    "model": "",
    "name": "电暖温控器(zv0.0.2)",
    "product_id": "7rghpoxo",
    "product_name": "电暖温控器(zv0.0.2)",
    "sub": true,
    "time_zone": "+01:00",
    "update_time": 1734247658,
    "uuid": "a484421345b3d"
  },
  "success": true,
  "t": 1735571133472,
  "tid": "84595286c6bf11eff8af778c0bd36"
}

</div></div></div></div></div></div></div>
</body>
</html>
![DeveicePhoto](https://github.com/user-attachments/assets/13f6c844-d7de-4754-90c4-ab5c6442149f)

**Comment by kapitonovdv** [comment]
Hi. I have the same device. It is currently not supported.
![image](https://github.com/user-attachments/assets/73b6456d-7a36-4daa-8c61-c9a792467a00)
![image](https://github.com/user-attachments/assets/f9d2437e-df80-4096-9fda-953321eae56a)

**Comment by DaeMonSxy** [comment]
Same here,
any success pairing with z2m?
